source/fe/fe_q_base.cc

```diff
@@ -172,10 +178,10 @@ struct FE_Q_Base<POLY,xdim,xspacedim>::Implementation
     // Add midpoint
     constraint_points.push_back (Point<dim-1> (0.5));
 
-    if (fe.degree>1)
+    if (q_deg>1)
```

Comment:
I am not sure, but it is worth checking whether FETools::compute_face_embedding... can replace this function. That's what BDM uses.